modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -0,0 +1,73 @@
+= Using MUnit Coverage in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can run a coverage report from Anypoint Studio.
```

Comment:
What value does the coverage report have for the user? What does it include?

Reply:
Included! Thanks!